eZ/Bundle/EzPublishCoreBundle/EventListener/RequestEventListener.php

```diff
@@ -82,6 +82,9 @@ public function onKernelRequestForward(GetResponseEvent $event)
                     $request->getContent()
                 );
                 $forwardRequest->attributes->add($request->attributes->all());
+                if ($request->headers->has('X-User-Hash')) {
+                    $forwardRequest->headers->set('X-User-Hash', $request->headers->get('X-User-Hash'));
+                }
```

Comment:
In FOSHttpCache/src/SymfonyCache/UserContextSubscriber can we make 'options' protected so these vales can be grabbed from them in extended classes?